### PR TITLE
Add CI Integration section + MegaLinter in README :)

### PR DESCRIPTION
**Pull request by nvuillam**
All is in the title :)

## Checklist
Not all of these might apply to your change but the more you are able to check
the easier it will be to get your contribution merged.

- [x] CI passes
- [x] Description of proposed change
- [x] Documentation (README, docs/, man pages) is updated
- [ ] Existing issue is referenced if there is one
- [ ] Unit tests for the proposed change

### CODE REVIEW

**Comment by abitrolly** [comment]
I guess you might interested to support this project more than others. If that's correct, then I bet you want to be aware of  https://github.com/mrtazz/checkmake/issues/44

**Comment by nvuillam** [comment]
@abitrolly I'm already drowning with MegaLinter maintenance :)

But if MegaLinter users report issues or improvement ideas about CheckMake, i'll make sure to redirect them here :)

**Comment by nvuillam** [comment]
Ox Security is a startup and is already founding..... MegaLinter (so me !) , so open-source support is already in their expenses :D ( MegaLinter was an independent project before ^^ )

**Comment by abitrolly** [comment]
Right. This is what I am saying. Your investors want to be sure that your dependencies will not fall apart once you are out in the market.

File: README.md [hunk 1]
```diff
@@ -71,6 +71,14 @@ cd checkmake
 make
 ```
 
+## Use in CI
+
+### MegaLinter
+
+checkmake is [natively embedded](https://oxsecurity.github.io/megalinter/latest/descriptors/makefile_checkmake/) within [MegaLinter](https://github.com/oxsecurity/megalinter)
+
+The install it, run `npx mega-linter-runner --install` (requires Node.js)
```
Comment (rasa) [inline README.md:80]: Shouldn't this be: *To* install it?

Comment (nvuillam) [reply README.md:80]: indeed, sorry for the typo :)

Comment (nvuillam) [reply README.md]: Fixed !

**Comment by mrtazz** [comment]
thanks for the contribution!

**Pull request merged**